vault/resource_quota_rate_limit.go

```diff
@@ -60,6 +73,14 @@ func quotaRateLimitCreate(d *schema.ResourceData, meta interface{}) error {
 	data["path"] = d.Get("path").(string)
 	data["rate"] = d.Get("rate").(float64)
 
+	if intervalRaw, ok := d.GetOk("interval"); ok {
+		data["interval"] = intervalRaw.(int)
+	}
+
+	if blockIntervalRaw, ok := d.GetOk("block_interval"); ok {
+		data["block_interval"] = blockIntervalRaw.(int)
+	}
+
```

Comment:
Should be no need for the type assert when setting the value below. The schema's type is validated should match what Vault expects.

```suggestion
	if v, ok := d.GetOk("interval"); ok {
		data["interval"] = v
	}

	if v, ok := d.GetOk("block_interval"); ok {
		data["block_interval"] = v
	}

```